CHANGELOG.md

```diff
@@ -1,3 +1,243 @@
+
+* Drops `citus.single_shard_commit_protocol` GUC, defaults to 2PC
+
+* Drops support for `citus.multi_shard_commit_protocol`, always use 2PC
+
+* Enables re-partition joins in transaction blocks
+
+* Enables re-partition joins on followers
+
+* Enables shard replication > 1 hash distributed tables on Citus MX
+
+* Errors out if object has circular dependency
+
+* Errors out if object has dependency to an object with unsupported type
+
+* Fixes `ALTER STATISTICS IF EXISTS` bug
+
+* Fixes a bug preventing issues while create dependencies from multiple sessions
+
+* Fixes a bug that causes reading columnar metapage as all-zeros under high
+  write concurrency
+
+* Fixes a bug that could break `DROP SCHEMA/EXTENSON` commands when there is a
+  columnar table
+
+* Fixes a bug that could break pg upgrades due to missing `pg_depend` records
+  for columnar table access method
+
+* Fixes a bug that could cause `CREATE INDEX` to fail for expressions when using
+  custom `search_path`
+
+* Fixes a bug that could cause `worker_save_query_explain_analyze` to fail on
+  custom types
+
+* Fixes a bug that could cause failed re-partition joins to leak result tables
+
+* Fixes a bug that could cause prerequisite `columnarAM` objects being not
+  created during pg upgrade
+
+* Fixes a bug that could cause re-partition joins involving local shards to fail
+
+* Fixes a bug that limits usage of sequences in non-int columns
+
+* Fixes a bug that prevents `DROP SCHEMA CASCADE`
+
+* Fixes a clog lookup failure that could occur when writing to a columnar table
+
+* Fixes a crash that occurs when the aggregate that cannot be pushed-down
+  returns empty result from a worker
+
+* Fixes a missing `FROM` clause entry error
+
+* Fixes a possible segfault that could happen when reporting distributed
+  deadlock
+
+* Fixes an issue that could cause unexpected errors when there is an in-progress
+  write to a columnar table
+
+* Fixes an unexpected error that occurs due to aborted writes to a columnar
+  table with an index
+
+* Fixes an unexpected error that occurs when writing to a columnar table created
+  in older version
+
+* Fixes function signature generation bug
+
+* Fixes issue when compiling Citus from source with some compilers
+
+* Fixes issues on `ATTACH PARTITION` logic
+
+* Fixes naming issues of newly created partitioned indexes
+
+* Implements `TEXT SEARCH CONFIGURATION` propagation
+
+* Implements `TEXT SEARCH DICTIONARY` propagation
+
+* Improves concurrent metadata syncing and metadata changing DDL operations
+
+* Improves self-deadlock prevention for `CREATE INDEX / REINDEX CONCURRENTLY`
+  commands for builds using PG14 or higher
+
+* Introduces `citus_backend_gpid()`
+
+* Introduces `citus_check_cluster_node_health` UDF to check cluster connectivity
+
+* Introduces `citus_check_connection_to_node` UDF to check node connectivity
+
+* Introduces `citus_coordinator_nodeid` UDF
+
+* Introduces `citus_stat_activity` view
```

Comment:
I am more inclined to start with the positive message here.

How about the following (split into multiple lines)

    Introduces `citus_stat_activity` view and drops `citus_worker_stat_activity` UDF

Reply:
I'm not sure if we can say `instead` here. I don't think they are completely interchangeable.
Also they both have a UDF and a view. If we have to call them something I think `view` is better since that's the part users use.